lib/mix/lib/mix/tasks/escript.build.ex

```diff
@@ -163,7 +163,17 @@ defmodule Mix.Tasks.Escript.Build do
     end
 
     app = Keyword.get(escript_opts, :app, project[:app])
-    strip_beam? = Keyword.get(escript_opts, :strip_beam, true)
+
+    # Need to keep :strip_beam option for backwards compatibility so
```

Comment:
```suggestion
    # Need to keep :strip_beam option for backward compatibility so
```
Minor nitpick. Backwards is an adverb, backward is the adjective

Reply:
Touché. Lol. I’ll change it

Reply:
Changed